pkg/parser/expression/parser_help_test.go

```diff
@@ -0,0 +1,118 @@
+package expression
+
+import (
+//	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path"
+//	"testing"
+
+	"github.com/gofed/symbols-extractor/pkg/parser/alloctable"
+	"github.com/gofed/symbols-extractor/pkg/parser/symboltable/stack"
+	typeparser "github.com/gofed/symbols-extractor/pkg/parser/type"
+	"github.com/gofed/symbols-extractor/pkg/parser/types"
+	gotypes "github.com/gofed/symbols-extractor/pkg/types"
+)
+/**** HELP FUNCTIONS ****/
+
+func prepareParser(pkgName string) *types.Config {
+	c := &types.Config{
+		PackageName:           pkgName,
+		SymbolTable:           stack.New(),
+		AllocatedSymbolsTable: alloctable.New(),
+	}
+
+	c.SymbolTable.Push()
+	c.TypeParser = typeparser.New(c)
+	c.ExprParser = New(c)
+
+	return c
+}
+
+func getAst(gopkg, filename string, gocode interface{}) (*ast.File, *token.FileSet, error) {
+	fset := token.NewFileSet()
+	gofile := path.Join(os.Getenv("GOPATH"), "src", gopkg, filename)
+	f, err := parser.ParseFile(fset, gofile, gocode, 0)
```

Comment:
As I see that's not true. `fset` ise returned by the getAst function. Originaly it wasn't planned, but I found it useful later because of `ast.Print`. When tests will be complete, it can be probably dropped from function, but now it is useful sometimes when you want to see the AST.